nntrainer/layers/layer.cpp

```diff
@@ -415,4 +416,12 @@ void Layer::print(std::ostream &out, unsigned int flags) {
   }
 };
 
+std::shared_ptr<Layer> getLayerDevel(std::shared_ptr<ml::train::Layer> l) {
+  std::shared_ptr<LayerNode> lnode = std::static_pointer_cast<LayerNode>(l);
+
+  std::shared_ptr<Layer> &layer = lnode->getObject();
+
+  return layer;
```

Comment:
I guess we could reduce some copies by making this one liner

```suggestion
   return std::static_pointer_cast<LayerNode>(l)->getObject();
```

Reply:
yes, will do.

Reply:
done.